tests/network/connectivity/test_ovs_linux_bridge.py

```diff
@@ -50,6 +50,7 @@ def test_ipv4_linux_bridge(
     @pytest.mark.post_upgrade
     @pytest.mark.polarion("CNV-11125")
     @pytest.mark.ipv6
+    @pytest.mark.jira("CNV-58530", run=True)
```

Comment:
Shouldn't it be `run=False`?

Reply:
No, I'll explain:
IPv6 is supported on bare-metal, so we want these tests to run on these clusters and expect them to pass.
On virtual clusters (PSI) IPv6 is not supported, so marking these tests with `jira` and let them run means xailing them.

Reply:
so since IPv6 can't run on PSI, you'll need to remove `gating` marker here (as gating is ran on PSI only now)

Reply:
> so since IPv6 can't run on PSI, you'll need to remove `gating` marker here (as gating is ran on PSI only now)

@vsibirsk IPv6 support becomes a must, so that's why we issued https://issues.redhat.com/browse/CNV-56595 to limit gating tests only for bare-metals. You can go over the issue description and comments to understand why running gating only on PSI is no longer legit (if something is not clear, please reach out to me and I'll happily explain).
Just like this PR was [merged in main](https://github.com/RedHatQE/openshift-virtualization-tests/pull/586/files), it can and should be merged in 4.18 as well.
Thank you

Reply:
ack. wasn't aware of that network task. thanks for the clarification @yossisegev